app/controllers/groups_controller.rb

```diff
@@ -36,11 +36,11 @@ def create
 
   def member_listing
     @group = Group.friendly.find(params[:id])
-    @members = @group.users
+    user_ids = @group.user_groups.pluck(:user_id)
+    @members = User.where(id: user_ids)
```

Comment:
Should also abstract the above two lines into the `Group` model.
```ruby
@members = @group.members()
```
All the business logic dealing with `UserGroup` should be contained in the `Group` model